pages/Wishlist/wishlist.css

```diff
@@ -0,0 +1,17 @@
+@import url('../../resource/style.css');
+@import url('../../resource/variables.css');
+
+/* Cross icon */
+.card-cross {
+    cursor: pointer;
```

Comment:
You can make utility class for such one liners.

Reply:
Thankyou for your comments. I have made the changes